sprint 4

irinaBerendeeva87 opened the pull request.

irinaBerendeeva87 committed:
project

nikolay-977 commented on src/main/java/dropdownText/HomePageScooter.java:
```diff
@@ -0,0 +1,26 @@
+
+    //  локатор для dropdown
+    public Boolean isPanelVisible(int index) {
+        By locator = By.id("accordion__panel-" + index);
```
Патерн Page Object включает веб-элементы и шаги (действия или методы взаимодействия между веб-элементами для получения актуального результата). Опиши все веб-элементы, которые используются шагах, с помощью локаторов и укажи модификатор доступа private. Это позволит переиспользовать веб-элементы в разных шагах(методах), а шаги в разных тестах.

nikolay-977 commented on src/main/java/dropdownText/HomePageScooter.java:
```diff
@@ -0,0 +1,26 @@
+package dropdownText;
```
Исправь наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы.

nikolay-977 commented on src/test/java/OrderScooterTest.java:
```diff
@@ -0,0 +1,83 @@
+        this.phonenumber = phonenumber;
+    }
+
+    @Parameterized.Parameters  // добавили аннотацию
```
В параметризованных тестах для аннотации @Parameterized.Parameters лучше использовать аргумент name: @Parameterized.Parameters(name = "Тестовые данные: {0} {1}"), где {0}, {1} - индексы параметров. Это повысит информативность проверки.

nikolay-977 commented on src/test/java/OrderScooterTest.java:
```diff
@@ -0,0 +1,83 @@
+        objOrderScooter.clickYesInDialog();
+        objOrderScooter.isPanelVisible();
+    }
+
```
Code Style. Убери лишние пропуски строк. Для зрительного разделения кода достаточно использовать один пропуск строки. Между фигурными скобками пропуски не нужны. Пропуски строк должны быть однообразными.

irinaBerendeeva87 committed:
changes:
- added locators;
- change Package name;
- deleted space;
- added name in @Parameterized.Parameters
